platform/security/idp/security-idp-client/src/main/resources/templates/submitForm.handlebars

```diff
@@ -12,28 +11,26 @@
  *
  **/
 -->
-<!DOCTYPE html
-        PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
-        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+<!DOCTYPE html>
 <head>
+    <meta charset="UTF-8">
```

Comment:
Use this `<meta http-equiv="Content-Type" content="text/html; charset=utf-8">`

Reply:
Done